### DISCUSSION TITLE
Upgrade Cost Question

### DISCUSSION THREAD

**Discussion by stonstad** [visualdesigncafe/nature-renderer#66] [General]
Hi there. What is the cost to upgrade from 2022 to 6.0?  I'm on Unity 2022 and plan to purchase NR. I intend to use 6.0 once it is generally available. I'd like to avoid a duplicate purchase of NR this year.

**Comment by stonstad** [comment]
Amazing.  No response from NR.  I guess they don't want my money.

**Reply by krazyxiii** [reply]
Wow. I have the same question.

**Comment by zertrax** [comment]
I own 2022 and I can see the new version priced at $50 with a 75% discount, this will most likely apply to you as well

**Comment by krazyxiii** [comment]
Oh thanks. I see that too. I guess I posted under the wrong forum. I mean
Nature Shaders.

**Comment by stonstad** [comment]
If NR is not working for you, or you like actual support, you might take a look at GPU Instancer Pro. It works for trees, grass, and prefabs.  I found it to work much better than NR.
